Projects/COM/COM04/COM04-AWS200.md

```diff
@@ -0,0 +1,53 @@
+# COM04-AWS200 - Create a cluster of virtual machines using docker swarm
+
+## Cloud Service Provider
+
+- Amazon Web Services
+
+## Difficulty
+
+- Level 200 (Intermediate)
+
+## Project's Author(s)
+
+[Karan Gauswami](https://github.com/KaranGauswami)
+
+## Objectives
+
+### You need to complete the following:
+
+- Create a VPC for our EC2 Instances
```

Comment:
Why is a new VPC needed? There is a VPC in place by default on AWS. There is no harm in using the default VPC for this. Projects should overlap only when really needed (COM-NET). New VPC means new route tables, new IGW and new Subnets all have to be created manually.

Reply:
Removed VPC